context/app/static/js/components/Search/styleForSidebar.js

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+import { withStyles } from '@material-ui/core/styles';
+
+// In the latest version, "ExpansionPanel" is renamed to "Accordion".
+import AccordionDetails from '@material-ui/core/ExpansionPanelDetails';
+import AccordionSummary from '@material-ui/core/ExpansionPanelSummary';
+import Accordion from '@material-ui/core/ExpansionPanel';
+
+import { SideBar } from 'searchkit';
+
+const InnerAccordion = styled(Accordion)`
+  box-shadow: none;
+  border: none;
+  margin: 0 !important; // Override margin on expand.
```

Comment:
Was the `!important` needed? Styled-component styles should override mui styles.

Reply:
With `!important`:
<img width="217" alt="Screen Shot 2020-09-11 at 4 43 51 PM" src="https://user-images.githubusercontent.com/730388/92971233-15e24c00-f44e-11ea-8324-75fe7a7f4fc0.png">
without:
<img width="206" alt="Screen Shot 2020-09-11 at 4 44 18 PM" src="https://user-images.githubusercontent.com/730388/92971248-1f6bb400-f44e-11ea-8aa8-e3ca1ec9bb93.png">

The extra margin is coming from:
```
.portal-MuiExpansionPanel-root-132.portal-MuiExpansionPanel-expanded-134 {
    margin: 16px 0;
}
```
I think that's taking precedence because it's two classes, instead of just one? I could supply `margin: 0` for root and expanded... but that felt like a hassle.